jenkins-scripts/dsl/_configs_/OSRFUNIXBase.groovy

```diff
@@ -53,11 +53,12 @@ class OSRFUNIXBase extends OSRFBase
                   def node = build.getBuiltOn()
                   def old_labels = node.getLabelString()
 
-                  println("# BEGIN SECTION: NVIDIA MISMATCH RECOVERY")
```

Comment:
(nit) I think is worthwhile to add a section to say that it's a postbuild recovery process. Maybe something in the lines of `#BEGIN SECTION: ATTEMPTING TO RECOVER FROM FAILURE`

Reply:
I moved down the `#BEGIN SECTION` as it would make sense that the section is "recovering process" instead of "checking if I can do a recovery process".